doc/content/devices/generate-qr-code/_index.md

```diff
@@ -40,27 +40,33 @@ ttn-lw-cli end-devices list-qr-formats
 ```
 </details>
 
-The formats show the fields of the end device that are used in the QR code.
+The `paths` object from the output shows the end device fields that are used to generate the QR code. 
 
-## Generate QR Code for Identification
+## Generate an Identification QR Code
 
-To generate a QR code for identification:
+To generate a QR code to help you easily identify your device:
 
 ```
 ttn-lw-cli end-devices generate-qr <application-id> <device-id> --format-id <qr-code-format>
 ```
 
+The generated QR code will automatically be saved to the current directory with the device ID as a file name, in PNG format with a default size of 300 pixels. Use the `--folder` and `--size` flags to change the save location and image size.
+
 <details><summary>Example of a generated identification QR code</summary>
 
 {{< figure src="qr-identification.png" alt="Device QR Code for Identification" >}}
 
 </details>
 
-This saves the QR code to the current directory with the device ID as file name, in PNG format with a default size of 300 pixels. Use `--folder` and `--size` to change the save location and image size.
+If you scan the generated identification QR code, you will see that it stores the information in the following format: `LW:<schema-id>:<join-eui>:<dev-eui>:<claim-authentication-code>`. The SchemaID value will be `D0` as it is the only device schema version described by the [LoRaWAN Device Identification QR Codes for Automated Onboarding Technical Recommendation (TR005)](https://lora-alliance.org/wp-content/uploads/2020/11/TR005_LoRaWAN_Device_Identification_QR_Codes.pdf). For devices that haven't been [made claimable]({{< relref "../device-claiming/make-device-claimable" >}}), the claim authentication code will be `00000000`.
```

Comment:
Where it's stated `<claim-authentication-code>` and blank value `00000000`, it is in fact the vendor and the profile ID. Both are 16-bit, and in hex that is 8 hex numerals.

The claim authentication code can be part of the QR code if it is set. If it's not set, it's not there. See the referenced PDF for additional info.